core/src/Kokkos_View.hpp

```diff
@@ -1963,14 +1962,23 @@ using DeducedCommonPropsType =
     typename Impl::DeduceCommonViewAllocProp<Views...>::prop_type;
 
 // User function
+// This function is required for example for dynamic length embedded ensemble
+// types It is used to propagate necessary information (like the ensemble size)
+// Most of the time it is called with a single view however.
+// Furthermore, the propagated information is not just for view allocations.
+// From what I can tell this type of functionality is the equivalent to
```

Comment:
I think it is "the equivalent of" or "equivalent to"

Reply:
ah I saw you approved anyway, so I am not gonna change that :-D